src/app/counter/counter.component.ts

```diff
@@ -11,5 +11,9 @@ export class CounterComponent implements OnInit {
 
   ngOnInit(): void {
   }
+  count:number = 0;
```

Comment:
You can initialize the value of the 'count' variable in the ngOnInit() method.

Reply:
okay